pkg/controller/controller.go

```diff
@@ -135,7 +135,7 @@ func NewUnmanaged(name string, mgr manager.Manager, options Options) (Controller
 		CacheSyncTimeout:        options.CacheSyncTimeout,
 		SetFields:               mgr.SetFields,
 		Name:                    name,
-		Log:                     options.Log.WithName("controller").WithName(name),
+		Log:                     options.Log.WithValues("controller", name),
```

Comment:
@alvaroaleman 
Maybe we can set the name and k/v?

I just saw this user story in the kubernetes contextual logging KEP:
> When kube-scheduler starts processing a pod, it creates a new logger with logger.WithValue("pod", klogr.KObj(pod)) and passes that around. While iterating over plugins in certain operations, another logger gets created with logger.WithName(<operation>).WithName(<plugin name>) and then is used when invoking that plugin. This adds a prefix to each log message which represents the call path to the specific log message, like for example NominatedPods/Filter/VolumeBinding.
> https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging#story-1

To me it seemed strange to have a msg prefix with the controller. But looking at that user story it kind of makes sense.

Reply:
Yeah, that makes sense and seems a good idea